compiler/src/iree/compiler/Codegen/Common/GPU/test/CMakeLists.txt

```diff
@@ -26,7 +26,8 @@ iree_lit_test_suite(
     "gpu_greedily_distribute_to_threads.mlir"
     "gpu_infer_memory_space.mlir"
     "gpu_lower_to_ukernels.mlir"
-    "gpu_materialize_encoding.mlir"
+    "gpu_materialize_encoding_cdna3.mlir"
+    "gpu_materialize_encoding_rdna3.mlir"
```

Comment:
Can we instead name these files based on the target we use for testing? I know that it's really cdna3/rdna3, but we usually pass a target in the test file.